api/llm.py

```
@@ -199,15 +199,13 @@ def _define_ontology() -> Ontology:
 def _create_kg_agent(repo_name: str):
     global ontology
 
-    openapi_model    = OpenAiGenerativeModel("gpt-4o")
-    #gemini_model     = GeminiGenerativeModel("gemini-1.5-flash-001")
-    #gemini_model_pro = GeminiGenerativeModel("gemini-1.5-pro")
+    model = LiteModel(model_name="gemini/gemini-2.0-flash-exp")
```

Ruff (0.8.2)

Undefined name `LiteModel`

(F821)